### PR TITLE
Plasma vessels regenerate slowly off resin

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
balance: Plasma vessels, the organs inside aliens that are responsible
for their internal plasma storage, will regenerate small amounts of
plasma even if the owner is not on resin weeds.
/:cl:

- Rate is 10% of normal. Stops aliens accidentally spending their plasma
poorly, and then being unable to plant weeds ever again. Breathing in
plasma is not really an option for most aliens, and not many people know
about it.
- This is an "anti-stupidity" feature, in essence.

### CODE REVIEW

**Comment by Incoming5643** [comment]
You can also just stand in front of the PA to regenerate plasma, but that's even more obscure.

**Comment by coiax** [comment]
I did not know that. Also, the PA is not really used anymore since we got the SM.

**Comment by WJohn** [comment]
For anyone thinking this will make xenos even more powerful:

In the case of a xeno infestation: There are already weeds everywhere, plasma is in ABSURD abundance, this does nothing to add to it.
In the case of a xeno having screwed itself over, having used too much plasma to plant weeds: This can save some potential xeno infestations from just dying out to early due to incompetence, rare as it is.
In the case of a contained xeno in a small room with no vents and bolted and out of plasma and is NOT a hunter: It's a very minor buff, sure. It means they'll eventually be able to melt their way out if they're SOMEHOW not killed in the time it takes to generate 50 plasma at an abysmal rate. This scenario is *ridiculously* rare, though, so it barely matters.

TL;DR this helps prevent stupid incompetent 1st time xenos from screwing themselves over really early.

**Comment by bgobandit** [comment]
it would be cool for humans to be able to harvest plasma from xeno weeds

**Comment by XDTM** [comment]
i really would've liked this when i was a queen drifting in space with no plasma

it was very embarassing

**Comment by Core0verload** [comment]
Actually, standing in front of SM should work too.

**Comment by PKPenguin321** [comment]
>being the first larva and not ventcrawling into the atmos plasma storage area to evolve straight to full queen instantly

get good

**Comment by Kromgar** [comment]
@PKPenguin321 Plasma doesn't allow you to instantly evolve to queen from larva

**Comment by PKPenguin321** [comment]
the atmos plasma area has so much pressure it fills your plasma reserves instantly so pretty much

**Comment by Kromgar** [comment]
@PKPenguin321 larva don't have plasma sacs they just have a counter that increases faster only if your on resin

**Comment by PKPenguin321** [comment]
ok you still have to wait for the first larva evolution whatever

**Comment by coiax** [comment]
Actually larva do have plasma glands, but their growth is independent of that.
